discovery/openstack/mock_test.go

```diff
@@ -583,6 +583,6 @@ func (m *SDMock) HandleFloatingIPListSuccessfully() {
 		testHeader(m.t, r, "X-Auth-Token", tokenID)
 
 		w.Header().Add("Content-Type", "application/json")
-		fmt.Fprintf(w, listOutput)
+		fmt.Fprint(w, listOutput)
```

Comment:
I think you need to rebase.

Reply:
I have rebased now, but the problem fixed here (as marked by staticcheck) is currently in master.

Reply:
Weird, @simonpasquier did you not fix these already?

Reply:
I suspect that @beorn7 runs `staticcheck` from master which introduces those new checks. In the CI, we always run a specific version of `staticcheck` (currently 2019.1).

Reply:
Grand, not stepping on anyone else's toes then.